packages/side-nav/package.json

```diff
@@ -0,0 +1,19 @@
+
+  {
+    "name": "@leafygreen-ui/side-nav",
+    "version": "0.9.0",
+    "description": "leafyGreen UI Kit Side Nav",
+    "browser": "./dist/index.web.js",
+    "main": "./dist/index.node.js",
+    "types": "./dist/index.d.ts",
+    "scripts": {
+      "build": "../../node_modules/.bin/webpack --config ../../webpack.config.js"
+    },
+    "license": "Apache-2.0",
+    "publishConfig": {
+      "access": "public"
+    },
+    "devDependencies": {
+      "@leafygreen-ui/lib": "^3.2.0"
```

Comment:
I have to update this in the boilerplate, but please use lib at 4.0.0

Reply:
also add the rest of the devDeps (menu, palette, emotion, etc.)

Reply:
@brookescarlett Is there a reason it's in the boilerplate? I don't think every component uses this.

Reply:
^ yes, but can make a separate PR quickly to update